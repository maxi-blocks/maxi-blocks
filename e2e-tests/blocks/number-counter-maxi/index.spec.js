/**
 * WordPress dependencies
 */
import { createNewPost } from '@wordpress/e2e-test-utils';

/**
 * Internal dependencies
 */
import {
	addCustomCSS,
	getBlockStyle,
	getEditedPostContent,
	insertMaxiBlock,
	updateAllBlockUniqueIds,
} from '../../utils';

describe('Number Counter Maxi', () => {
	it('Number Counter Maxi does not break', async () => {
		await createNewPost();
		await insertMaxiBlock(page, 'Number Counter Maxi');

		await updateAllBlockUniqueIds(page);

		expect(await getEditedPostContent(page)).toMatchSnapshot();
		expect(await getBlockStyle(page)).toMatchSnapshot();
	});
	it('Number Counter Custom CSS', async () => {
		await expect(await addCustomCSS(page)).toMatchSnapshot();
	}, 500000);
});
