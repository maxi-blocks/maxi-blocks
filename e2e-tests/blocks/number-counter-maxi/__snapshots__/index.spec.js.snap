// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Number Counter Maxi Number Counter Custom CSS 1`] = `"<style>body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u{ justify-content: center; background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u{ justify-content: center; background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box{ width: 250px;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box{ width: 250px;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle{ stroke: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle{ stroke: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background{ stroke: rgba(var(--maxi-light-color-2,242,249,253),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background{ stroke: rgba(var(--maxi-light-color-2,242,249,253),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text{ font-size: 40px; font-family: Roboto; color: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text{ font-size: 40px; font-family: Roboto; color: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan{ font-size: 26.666666666666668px;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan{ font-size: 26.666666666666668px;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text{ background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::before{ background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::before{ background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u::after{ background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__text::after{ background: red;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle{ background: red;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box__circle{ background: red;}</style>"`;

exports[`Number Counter Maxi Number Counter Maxi does not break 1`] = `
"<!-- wp:maxi-blocks/number-counter-maxi {"maxi-version-current":"1.4","maxi-version-origin":"1.4","blockStyle":"light","isFirstOnHierarchy":true,"uniqueID":"number-counter-maxi-1se8ef1z-u","customLabel":"Number-counter_1"} -->
<div class="wp-block-maxi-blocks-number-counter-maxi maxi-block maxi-nc-effect maxi-nc-effect-number-counter-maxi-1se8ef1z-u maxi-light number-counter-maxi-1se8ef1z-u maxi-number-counter-block" id="number-counter-maxi-1se8ef1z-u"><div class="maxi-number-counter__box"><div class="maxi-number-counter__box__container"><svg viewBox="0 0 185 185"><circle class="maxi-number-counter__box__background" stroke-width="5" fill="none" cx="92.5" cy="92.5" r="90"></circle><circle class="maxi-number-counter__box__circle" stroke-width="5" fill="none" cx="92.5" cy="92.5" r="90" stroke-linecap="" stroke-dasharray="0 565.4866776461628"></circle></svg><span class="maxi-number-counter__box__text"></span></div></div></div>
<!-- /wp:maxi-blocks/number-counter-maxi -->"
`;

exports[`Number Counter Maxi Number Counter Maxi does not break 2`] = `"<style>body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box{ width: 250px;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box{ width: 250px;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle{ stroke: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__circle{ stroke: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background{ stroke: rgba(var(--maxi-light-color-2,242,249,253),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__background{ stroke: rgba(var(--maxi-light-color-2,242,249,253),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text{ font-size: 40px; font-family: Roboto; color: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text{ font-size: 40px; font-family: Roboto; color: rgba(var(--maxi-light-color-4,255,74,23),1);}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan{ font-size: 26.666666666666668px;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xl"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="l"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="m"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="s"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive="xs"] .maxi-block.maxi-block--backend.number-counter-maxi-1se8ef1z-u .maxi-number-counter__box .maxi-number-counter__box__text tspan{ font-size: 26.666666666666668px;}</style>"`;
